domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/preparedness/domain/DomainAPI.kt

```diff
@@ -0,0 +1,56 @@
+package com.krzykrucz.elesson.currentlesson.preparedness.domain
+
```

Comment:
the name of this file `DomainAPI.kt` is yet another convention, so far, Types.kt` ,`Workflows.kt` have been used, any clue about it? I guess it might be helpful to sit together, review the whole project and think about its shape